### PR TITLE
Rewrites most of the commands and gives them better names

### PR DESCRIPTION
**Pull request by redmoogle**
This. was. straight. pain.

The commands have been given more friendly names along with their constructors being modified

to use a _probably_ better system thats not using positional arguments.

### CODE REVIEW

File: config/discord.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "discord_token": "notokenhere",
+  "discord_token": "ODE5OTQ4NzkxNzIwOTY4MjEy.YEuC0Q.KHhDS8yZQssW-YNMLUKFrg3d4fU",
```
Comment (ToasterBiome) [inline config/discord.json]: Should this be here?

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

**Comment by ToasterBiome** [comment]
well ya fixed it before I even got to comment LOL

**Comment by LoliconSlayer** [comment]
Please update the image commands to reflect the PR's that just got merged today

**Comment by redmoogle** [comment]
> Please update the image commands to reflect the PR's that just got merged today

I just forked it 2 hours ago

**Comment by AshCorr** [comment]
> > Please update the image commands to reflect the PR's that just got merged today
> 
> I just forked it 2 hours ago

conflicts gonna conflict.

**Comment by AshCorr** [comment]
Not really sure how renaming commands to `iSomething` is better, to me it just seems confusing.

**Comment by redmoogle** [comment]
> Not really sure how renaming commands to `iSomething` is better, to me it just seems confusing.

That was to group image commands together thing about img tho

**Comment by AshCorr** [comment]
Every other class in the codebase is in the format of `<ParentClassName><ChildClassName>` and every file has a a class with the same name as the file. Files also follow the CamelCase format where the first character of every word is capitalized.

In other languages prefixing a file with `I` usually indicates its an interface and not an implementation.

I think instead of just prefixing image commands with `i` you should try creating some kind of `DiscordCommandImage` class that all image commands can inherit from to share all of the image randomization & embedding logic.

**Comment by redmoogle** [comment]
> Every other class in the codebase is in the format of `<ParentClassName><ChildClassName>` and every file has a a class with the same name as the file. Files also follow the CamelCase format where the first character of every word is capitalized.
> 
> In other languages prefixing a file with `I` usually indicates its an interface and not an implementation.
> 
> I think instead of just prefixing image commands with `i` you should try creating some kind of `DiscordCommandImage` class that all image commands can inherit from to share all of the image randomization & embedding logic.

yea but what do I calls the files then just like regular commands? because I changed them to be not super long

**Comment by alexkar598** [comment]
Closing because:
1) Cornflaked
2) Theres like 3 different refactors in here
3) I don't see the point in half of them
4) Noone seems to want to merge this PR.

Feel free to fix your PR and find someone to review and merge it then get it reopened.

**Pull request closed without merging**